tests/test_common.py

```diff
@@ -1,9 +1,9 @@
-import unittest
 import tempfile
-import utils
-import os
+import unittest
 from os import path
-from common import *
+from recipemd.common import *
```

Comment:
Sorry, I had forgotten that it anyhow worked on jenkins. So I'd say we can just keep it knowing that, the only thing we could do is mention it in the readme in a contributor section.